app/views/shared/_footer.html.erb

```diff
@@ -66,7 +66,7 @@
           <p class="mb-4">
             <%= _t('nav.about_project').html_safe %>
           </p>
-          <p class="mb-0">&copy; 2018 <a href="quran.com">Quran.com</a>. <%= _t('nav.rights') %></p>
+          <p class="mb-0">&copy; 2020 <a href="quran.com">Quran.com</a>. <%= _t('nav.rights') %></p>
```

Comment:
please change this to `<%= Date.today.year %>`

Reply:
@naveed-ahmad Done :)